src/concepts/censuses.md

```diff
@@ -65,6 +66,8 @@ age, or update failures.
 
 ### DAU / Daily Active Users
 
+> **⚠** The mechanism described below is no longer used: see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted definition.
```

Comment:
I don't think this has changed.

Reply:
There's overlap between this and the definition in metrics for sure, but I think we need to make it clear that this isn't the currently accepted one. I'm open to changing the wording of this warning, but I think we should maintain it.

Reply:
```suggestion
> **⚠** This description of DAU is not authoritative; please see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the canonical definition.
```